plugins/grafana-custom-plugins/grafana-dependency-plugin/src/DependencyPanel.tsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
-import { PanelProps } from '@grafana/data';
-import { DependencyOptions } from 'types';
 import mermaid from 'mermaid';
+import { DependencyOptions } from 'types';
+import { PanelProps } from '@grafana/data';
 
 interface Props extends PanelProps<DependencyOptions> {}
 
 mermaid.initialize({
-  startOnLoad: true
+  startOnLoad: true,
```

Comment:
could you update the screenshots in documentations(plugin README and network-flow-visibility.md)?

Reply:
Yeah I'll upload the screenshots, I just wanted to get feedback on the colors beforehand in case I need to update them.

Reply:
<img width="515" alt="Screen Shot 2023-03-27 at 8 07 40 PM" src="https://user-images.githubusercontent.com/10016630/228117684-7aa78f66-4980-4cd4-8f6e-970ef4cbcfb6.png">

Reply:
thanks, could you also take a screenshot of the diagram when Grafana theme is set to dark?

Reply:
yeah, should I upload both to network-flow-visibility.md?

Reply:
just here please

Reply:
Hi @Dhruv-J  I think previously we have the color for text box background as something like this ![#dee6ff](https://placehold.co/15x15/dee6ff/dee6ff.png) instead of the blue in your screenshot. I feel that blue makes the black text a little bit difficult to read, could you change it back to the previous color or another color provide better readability?

Reply:
Yeah there's a lighter yellow orange I can change it to.

Reply:
<img width="500" alt="Screen Shot 2023-03-29 at 3 57 28 PM" src="https://user-images.githubusercontent.com/10016630/228686380-21f8493a-36d7-4d00-ad69-38802b6359e2.png">
<img width="488" alt="Screen Shot 2023-03-29 at 3 58 17 PM" src="https://user-images.githubusercontent.com/10016630/228686475-99a75c33-1aea-4b9c-b1e1-de539f14672c.png">
It's a bit better in light mode now, but the text color stays the same and fits surroundings, so the white on yellow is just a bit harder to read, but still readable I think.